rolling/distribution.yaml

```diff
@@ -404,6 +404,7 @@ repositories:
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/aws-gbp/aws_robomaker_small_warehouse_world-release.git
+      version: 1.0.5-1
```

Comment:
Bloom artifacts are specific to a target distribution and the release version being re-added here does not contain artifacts for Ubuntu Jammy.